xcp_d/workflow/bold.py

```diff
@@ -682,6 +719,14 @@ def init_boldpostprocess_wf(
         name='ds_report_qualitycontrol',
         run_without_submitting=False)
 
+    ds_report_confounds = pe.Node(DerivativesDataSink(
+        base_directory=output_dir,
+        suffix="design",
+        source_file=bold_file,
+        datatype="func"),
+        name='ds_report_confounds',
+        run_without_submitting=False)
```

Comment:
```suggestion
    ds_report_confounds = pe.Node(
        DerivativesDataSink(
            base_directory=output_dir,
            source_file=bold_file,
            dismiss_entities=["space", "den", "res"],
            datatype="func",            
            suffix="design",
            extension=".tsv",
        ),
        name="ds_report_confounds",
        run_without_submitting=False,
    )
```

I don't think the design matrix should have template-related entities. It should be agnostic to the data's space and resolution.

Why not put this in `init_writederivatives_wf`?

Reply:
I can try doing that instead